package.json

```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/ChristopheBougere/asl-validator#readme",
   "dependencies": {
     "ajv": "^8.12.0",
-    "asl-path-validator": "^0.14.2",
+    "asl-path-validator": "^0.15.0",
```

Comment:
new version supports variables in JSONPath expressions and also adds a new format type for `ResultPath` fields which are `ReferencePath` expressions but cannot contain variables.